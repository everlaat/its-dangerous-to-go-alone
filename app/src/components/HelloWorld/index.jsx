import React from 'react';
import styles from './style.scss';

const HelloWorld = () => (
  <div className={styles.root}>Hello World!</div>
);

export default HelloWorld;
