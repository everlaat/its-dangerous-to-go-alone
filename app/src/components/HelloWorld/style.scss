.root {
  font-weight: bold;
  color: red;
  margin: 40px auto 0 auto;
  border: 2px solid blue;
  padding: 30px;
  display: block;
  width: 200px;
  text-align: center;
  display: flex;
}
