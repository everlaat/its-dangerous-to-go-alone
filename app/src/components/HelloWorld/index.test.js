import React from 'react';
import HelloWorld from '../HelloWorld';

describe('HelloWorld', () => {
  test('renders correcly', () => {
    expect((<HelloWorld />)).toMatchSnapshot();
  });
});
