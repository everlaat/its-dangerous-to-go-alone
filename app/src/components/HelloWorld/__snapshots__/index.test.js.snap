// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld renders correcly 1`] = `<HelloWorld />`;
